grid/app/main/events/dfl/control_events.py

```diff
@@ -0,0 +1,71 @@
+import json
+from flask_login import login_user
```

Comment:
You should put a blank line between each group of imports.
You should group the third party imports together